acinclude/compiler-flags.m4

```diff
@@ -117,6 +113,17 @@ AC_DEFUN([SQUID_CC_GUESS_VARIANT], [
   ]) dnl AC_CACHE_CHECK
  ]) dnl AC_DEFUN
 
+dnl argument is a compiler flag. It will be attempted, and if suppported
```

Comment:
There are two arguments right now, not one. See https://github.com/squid-cache/squid/pull/667#pullrequestreview-718745265

Reply:
Resolved original issue